src/ray/gcs/redis_context.cc

```diff
@@ -637,14 +636,11 @@ Status RedisContext::Connect(const std::string &address,
   SetDisconnectCallback(redis_async_context_.get());
 
   // handle validation and primary connection for different types of redis
-  if (isRedisSentinel(*this)) {
+  if (isRedisSentinel()) {
     return ConnectRedisSentinel(*this, username, password, enable_ssl);
   } else {
```

Comment:
early exit? https://clang.llvm.org/extra/clang-tidy/checks/readability/else-after-return.html
so
```
if (IsRedis()) {
  ///
  return;
}
// no else here
```

Reply:
Same above, quite a few places apply

Reply:
I think this can be fixed in a separate PR